cmake/modules/dts.cmake

```diff
@@ -138,10 +134,9 @@ if(EXISTS ${DTS_SOURCE})
     list(APPEND DTS_SOURCE ${BOARD_DIR}/${BOARD}_${BOARD_REVISION_STRING}.overlay)
   endif()
 else()
-  # If we don't have a devicetree after all, there's not much to do.
-  set(header_template ${ZEPHYR_BASE}/misc/generated/generated_header.template)
-  zephyr_file_copy(${header_template} ${DEVICETREE_GENERATED_H} ONLY_IF_DIFFERENT)
-  return()
+  # If we don't have a devicetree, provide an empty stub
+  message(STATUS "Board does not provide a devicetree source, using empty stub file")
```

Comment:
Is this message needed ?

After all the `stub.dts` will also be printed here:
https://github.com/zephyrproject-rtos/zephyr/blob/8c6a3dbe37abd9b905ac89a375b97a811470dcbd/cmake/modules/dts.cmake#L182

as:
```
"-- Found BOARD.dts: <zephyr-base>/misc/stub.dts
```

so should be visible enough that a dts stub file is being used.

Reply:
I think you want to comment in https://github.com/zephyrproject-rtos/zephyr/pull/63799

Reply:
done